app/views/components/submit_button.scala.html

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  *@
 
-@(messageKey: String = "site.save_and_continue")(implicit messages: Messages)
+@(message: String = "site.save_and_continue")
```

Comment:
The default value here has not been changed.

Reply:
Sorry my miss - fixed in next commit